pythia8.sh

```diff
@@ -4,14 +4,24 @@ source: https://github.com/alisw/pythia8
 requires:
   - LHAPDF
   - HepMC
+  - boost
 tag: alice/v8210
 ---
 #!/bin/sh
 cd $SOURCEDIR
+
+# jkl: boost headers are required for LHAPDF6
 ./configure --prefix=$INSTALLROOT \
             --enable-shared \
             --with-hepmc2=${HEPMC_ROOT} \
-            --with-lhapdf6=${LHAPDF_ROOT}
+            --with-lhapdf6=${LHAPDF_ROOT} \
+	    --with-boost=${BOOST_ROOT}
+
+if [[ $ARCHITECTURE =~ "slc5.*" ]]; then
+    ln -s LHAPDF5.h include/Pythia8Plugins/LHAPDF5.cc
+    ln -s LHAPDF6.h include/Pythia8Plugins/LHAPDF6.cc
+    sed -i -e 's#\$(CXX) -x c++ \$< -o \$@ -c -MD -w -I\$(LHAPDF\$\*_INCLUDE) \$(CXX_COMMON)#\$(CXX) -x c++ \$(<:.h=.cc) -o \$@ -c -MD -w -I\$(LHAPDF\$\*_INCLUDE) \$(CXX_COMMON)#' Makefile
+fi
```

Comment:
It works with any half-way modern compiler, the problem really is that SLC 5 is so outdated and no one in the MC community is using and testing with such historic compilers.